### ISSUE TITLE
Truth Inspector improvements

### ISSUE THREAD

**Issue by Tisten** [OurMachinery/themachinery-public#1161]
A collection of suggestions to improve the usefulness of the awesome "Truth Inspector":
 - When doing "Inspect in Truth Inspector" from Entity tree and the inspector tab is already open behind other tab, bring the "Truth Inspector" tab to top.
 - When navigating with the "back" and "forward" buttons in the Inspector, go to the same tab as was used when you left the page. Otherwise it gets hard to recognize the view you arrive at. It would be nice to arrive at the same scroll location as well if the page had a scroll bar.
 - When viewing an entity object, there is a "Is an asset" field some ~13 lines down. And if this is "Yes" then there is an "Open in Asset Browser". It would be useful to 1. See the path to the asset and the name of it. 2. Have this "Open in Asset Browser" at the top together with the "Open in Property Tab" buttons. 3. It would also be great to be able to "Open entity in Entity Tree/Scene"
 - In Prototype Relationship view, please show UUID and owner for the entities as well. Perhaps even the path to the entity.
 
Overall, I often resort to doing "find in all files" of the UUID to find out which asset, and where in the asset some object of the truth recides in. So any info about where the data lives is a good addition. I guess this is for @simon-ourmachinery

**Comment by simon-ourmachinery** [comment]
Sounds all reasonable. I'll have a look at when I have the time slot free for the TT inspector

**Comment by simon-ourmachinery** [comment]
- [ ] I also miss "which file is this object stored in"
- [ ] Make it possible to search for UUID's

**Comment by simon-ourmachinery** [comment]
- [ ] > 3. It would also be great to be able to "Open entity in Entity Tree/Scene" check if doable without a lot of coupeling

**Comment by simon-ourmachinery** [comment]
@Tisten I am not sure what you mean with the subobject set inspection since you can see them. Maybe the UI / UX discoverbility is not good?
<img width="728" alt="image" src="https://user-images.githubusercontent.com/70658708/170683774-fb75d125-9b8c-4810-a90d-f8b962bd07d2.png">

**Comment by Tisten** [comment]
> @Tisten I am not sure what you mean with the subobject set inspection since you can see them. Maybe the UI / UX discoverbility is not good?

I have no idea how I did not see them. Thanks for pointing at them and un-blinding me.

**Comment by simon-ourmachinery** [comment]
No worries an idea how I can make this more obvious

**Comment by Tisten** [comment]
> No worries an idea how I can make this more obvious

No. Now when it is there, staring me in the face, I can't unsee it.

**Comment by Tisten** [comment]
I wonder a bit why "Inspect in Truth Inspector" isn't visible in the Entity Tree's context menu for components. I guess that is a question for @leonardo-ourmachinery since he wrote the code which excludes `tm_entity_tree_custom_context_menu_item_i` when no entity is selected.

**Comment by simon-ourmachinery** [comment]
@leonardo-ourmachinery  yeah told me that he would need to change some more things he just did not get around to doing that.

**Comment by leonardo-ourmachinery** [comment]
@Tisten @simon-ourmachinery I've fixed the "inspect in Truth inspector" for components here:
https://github.com/OurMachinery/themachinery/commit/b58b21d089a20161380c75222c2aaa125ef57899

and then here:
https://github.com/OurMachinery/themachinery/commit/ecb5013ca2f8bf663b67da2757da756fb1eb8fc0
